news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+
+
+## Issue Severity Criteria
+
+When a new issue is created, the first step is to understand what the problem is and then reproduce it. Once that work is done, the second step is to define the severity of that bug.
+
+Four severity levels have been defined, as explained below.
+
+
+### Critical
+
+The bug affects critical functionality or critical data and there is no workaround (no way to avoid it).
+A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
+Examples:
+
+- Difficulty accessing the front office or back office (significant slowdown, error during installation or update, fatal error)
+- Difficulty to globally manage categories, products or customers
+- Difficulty to globally place and manage orders
+
+A critical issue should result in a patch version that should be released as soon as possible. Our recent [spam attack](http://build.prestashop.com/news/fighting-against-spamming-again/) is a good example: it was a critical security issue as our users were flooded with fake customers’ accounts. As a consequence, a patch version has been released that resolve it on 1.6 and 1.7, a good example of a community work.
+
+
+### Major
+
+The bug affects major functionality or major data and there is a workaround (a temporary method for dealing with an issue until a more permanent solution is found), but it is not obvious or can be difficult to put in practice.
+
+A major issue affects a large percentage of users (> 30% and < 60%) and matches at least one of the following:
+
+- It impacts law compliance
+- It has a strong impact on the usability of the front-office / back-office or blocks another project
+- It is an important problem but not necessarily blocking the main activity of the seller
+
+Examples:
+
+- Being unable to add, configure or delete a theme or a module
+- Difficulty in operating a module properly
+- Impacts the price the customer pays
+
+### Minor
+
+The bug affects minor functionality or non-critical data and there is a reasonable workaround, even if it can be annoying when using your shop.
+
+
+Examples:
+
+- A tolerable slowdown
+- A display problem that prevents you to do something (you can’t click on an element)
+- An alert message displayed in your back-office that it is possible to close
+- The product doesn't get added to cart after deleting product customisation
+- When cloning a product, carriers information is not copied
+- Statistics graph not working or showing wrong data 
```

Comment:
> - The product doesn't get added to cart after deleting product customisation
> - When cloning a product, carriers information is not copied
> - Statistics graph not working or showing wrong data

Those three are too specific IMO, I'd delete them

Reply:
statistic graph would be a also good way to feed trolls. nice catch.

Reply:
I add them because otherwise there are only 3 examples and minor is the category where we have the most issues, I didn't find more generic examples